app/helpers/searches_helper.rb

```diff
@@ -26,4 +26,12 @@ def aggregation_count(aggregration, duration, buckets_pos)
     update_info = (duration == 30.days ? t("searches.show.month_update", count: count) : t("searches.show.week_update", count: count))
     link_to update_info, path, class: "t-link--black"
   end
+
+  def aggregation_bool_count(aggregration, field, filter)
+    count = aggregration["buckets"][field]["doc_count"]
+    return unless count > 0 && !params[:query].include?("#{field}:#{filter}")
```

Comment:
why we do need `!params[:query].include?("#{field}:#{filter}")`?

Reply:
Because if yanked filter is applied, we should not show `YANKED(123)`, because all the results are yanked only.

Reply:
>  we should not show YANKED(123)

we don't this for rest of the filters to keep it clear to the user that result shown is for the specific filter (not default result). imagine link being shared with the filter.

Reply:
If `yanked: true` filter is already applied by the user, without `!params[:query].include?("#{field}:#{filter}")` filter link will be visible and link will keep appending the same filter as
`search?query=name%3A+elasticsearch+++++yanked%3Atrue+AND+yanked%3Atrue`

Reply:
>  link will keep appending the same filter

this shouldn't affect user experience in any away. If you really care, you can probably update the url to use absolute path , I think `url_for` will avoid this.
I hope you agree that we should keep it clear to the user, whether the page is for default results or one with a filter. Please let me know if there is any confusion, I would be happy to explain further :slightly_smiling_face: